modules/swagger-models/src/main/java/io/swagger/models/parameters/Parameter.java

```diff
@@ -37,7 +37,6 @@
 
     void setReadOnly(Boolean readOnly);
 
-    Boolean getAllowEmptyValue();
 
-    void setAllowEmptyValue(Boolean allowEmptyValue);
+
```

Comment:
these 2 methods should also be moved to SerializableParameter interface

Reply:
SerializableParameter extends Parameter Interface, so there's no need to remove it here. It was being inherited from the Parameter Interface.

Reply:
Don't understand the comment, however the above comment applies here also:
We are moving the property from the more generic interface/abstract class `Parameter/AbstractParameter` which is also implemented by `BodyParameter`, to the less generic `SerializableParameter/AbstractSerializableParameter` which is not implemented by BodyParameter; this is implemented also by other parameters (not query or formData) but it's a good compromise of effort/result for this scope.

Reply:
Methods moved.